space/space.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/fabric8-services/fabric8-wit/gormsupport"
 	"github.com/fabric8-services/fabric8-wit/log"
 
+	"fmt"
```

Comment:
I know this is not important but the golang guide suggests to separate the blocks of imports into:

golang pkgs

our pkgs

external github pkgs